SU-HSDO/stanford_mrc
# MRCD8-246 footer styling

# READY FOR REVIEW

# Summary
- Fine-tuned the styling for the site footer
- Ensured that all is working at various breakpoints
- Based on reasonably small amount of lines of code, added to base with other footer styles

# Needed By (Date)
- End of sprint (3/1)

# Urgency
-N/A

# Steps to Test
1. Pull this branch
2. Ensure that footer mirrors mock in InVision (closely, may not be exact due to inherit and knowing changes) at https://invis.io/DCEGPUZS4#/272630312_Homepage_W-_Carousel

Note: the added code works with the Block Class module (with each footer block being given the decanter-width-one-fourth class). If Block Class is not configured or enabled, you can add the class through inspector to finalize test.

# Affected Projects or Products
- stanford_mrc
- MRC

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-246

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
